spec/models/about_spec.rb

```diff
@@ -28,6 +31,19 @@ def register_stat(name, stats_block)
       )
     end
 
+    it "uses the cache" do
+      ActiveRecord::Base.connection.query_cache.clear
+      ActiveRecord::Base.connection.enable_query_cache!
+      described_class.new.stats
+      expect(ActiveRecord::Base.connection.query_cache.length).not_to eq(0)
+
+      ActiveRecord::Base.connection.query_cache.clear
+      described_class.new.stats
+      expect(ActiveRecord::Base.connection.query_cache.length).to eq(0)
+    ensure
+      ActiveRecord::Base.connection.disable_query_cache!
```

Comment:
I think we can just use the `track_sql_queries` helper here.

```
initial_count = track_sql_queries do
  described_class.new.stats
end

final_count = track_sql_queries do
  described_class.new.stats
end

expect(final_count).to eq(initial_count)
```

Reply:
Ah, I was trying to grep for this, but couldn't find it/remember the name